priv/static/index.html

```diff
@@ -44,7 +44,14 @@
           <div class="col-md-2">
           </div>
           <div class="col-md-8">
-            <input type="text" id="siteSearch" class="input-lg" placeholder="Enter a website you love..." autofocus>
+            <input type="url" id="siteSearch" class="input-lg" placeholder="Enter a website you love..." list="defaultURLs" autofocus>
+
+            <datalist id="defaultURLs">
+              <option value="https://www.google.com/" />
+              <option value="https://www.amazon.com/" />
+              <option value="https://boardgamegeek.com" />
+            </datalist>
```

Comment:
this basically provides a couple of quick options for people in case they can't think of any. Any ideas for what we should include here? (or maybe you don't want to add this feature?)